### DISCUSSION TITLE
Feature Request - Doorbell Button Notification

### DISCUSSION THREAD

**Discussion by tdashmike** [themactep/thingino-firmware#387] [Ideas]
Ability to send a mqtt message when the doorbell button is pressed on the Wyze doorbells. Example attached, courtesy of juniper on Discord.

[overlay.zip](https://github.com/user-attachments/files/18220158/overlay.zip)

**Comment by vsviridov** [comment]
How would one use this? Do you just unpack it to `/overlay` and call it a day?
